front/src/applications/rollingStockEditor/components/RollingStockEditorButtons.tsx

```diff
@@ -38,7 +43,13 @@ function RollingStockEditorButtons({
             })
           );
         })
-        .catch(() => {
+        .catch((error) => {
+          console.warn('error delete ==>', error.data);
+          if (error.status === 409) {
+            console.warn(
+              `Ce MR est utilisé dans un scenario =>  Projet: ${error.data.context.usage[0].project_name}; scenario: ${error.data.context.usage[0].scenario_name}`
+            );
+          }
```

Comment:
You're not sure to get all this stuff in error.data. Use conditionnal chaining

Reply:
done